### ISSUE TITLE
bug: 🐝 

### ISSUE THREAD

**Issue by MrSeccubus** [JamesIves/github-pages-deploy-action#1699]
### Describe the bug

```
Run JamesIves/github-pages-deploy-action@v4
  with:
    folder: admin
    token: ***
    branch: gh-pages
    clean: true
    force: true
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.7.17/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.7.17/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.7.17/x64
    Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.7.17/x64
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.7.17/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.7.17/x64/lib
Error: File not found: '/home/runner/work/_actions/JamesIves/github-pages-deploy-action/v4/lib/main.js'
```

### Reproduction Steps

Deploy via the action. 
see https://github.com/DIVD-NL/web-www/actions/runs/11080665418/job/30794766954

### Logs

```shell
See bug
```

### Workflow

```yml
https://github.com/DIVD-NL/web-www/blob/main/.github/workflows/build_pages_native.yml
```

**Comment by limonte** [comment]
Same issue here https://github.com/sweetalert2/sweetalert2.github.io/actions/runs/11082516348/job/30795780781

**Comment by gismountains** [comment]
Workaround for now: https://github.com/JamesIves/github-pages-deploy-action/issues/1697#issuecomment-2379933098

**Comment by JamesIves** [comment]
This issue has now been resolved as of [v4.6.7](https://github.com/JamesIves/github-pages-deploy-action/releases/tag/v4.6.7), apologies about that. I'm going to make some adjustments to the integration tests so this doesn't go missed during the release cycle. The `v4` major tag has also been updated so any existing workflows should now work as expected.
